repositories/src/test/scala/cromwell/pipeline/datastorage/dao/repository/MarshallerTests.scala

```diff
@@ -0,0 +1,112 @@
+package cromwell.pipeline.datastorage.dao.repository
+
+import cromwell.pipeline.datastorage.dto._
+import cromwell.pipeline.datastorage.dto.auth.{ SignInRequest, SignUpRequest }
+import cromwell.pipeline.datastorage.dto.user.{ PasswordUpdateRequest, UserUpdateRequest }
+import org.scalatest.{ AsyncWordSpec, Matchers }
+import org.scalatestplus.scalacheck.ScalaCheckDrivenPropertyChecks
+import play.api.libs.json.{ JsError, JsSuccess }
+
+class MarshallerTests extends AsyncWordSpec with Matchers with ScalaCheckDrivenPropertyChecks {
+  import utils.ArbitraryUtils._
+  import utils.FormatUtils._
+
+  "MarshallerFormat" when {
+    "format UserUpdateRequest" in {
+      forAll { (a: UserUpdateRequest) =>
+        val parseResult: UserUpdateRequest = userUpdateRequestFormat.reads(userUpdateRequestFormat.writes(a)) match {
+          case JsSuccess(value, _) => value
+          case JsError(_)          => throw new RuntimeException("Could not parse request")
+        }
+        parseResult should equal(a)
```

Comment:
Thumbs up for ScalaCheck.
However, instead of throwing an exception in test, you must fail the test. For example, you can do this
```suggestion
        val parseResult: Option[UserUpdateRequest] = userUpdateRequestFormat.reads(userUpdateRequestFormat.writes(a)).asOpt
        parseResult should equal(Some(a))
```
Alternatively 
```suggestion
        val parseResult: UserUpdateRequest = userUpdateRequestFormat.reads(userUpdateRequestFormat.writes(a)) match {
          case JsSuccess(value, _) => value
          case JsError(_)          => fail("Could not parse request")
        }
        parseResult should equal(a)
```
Note that `fail` also throws an exception, but this is a special one and it is handled by scalatest.

Reply:
Fixed, thanks